### ISSUE TITLE
Segfault whan trying to play

### ISSUE THREAD

**Issue by rodrigoaguilera** [SimonKagstrom/despotify#1]
I'm wondering if despotify is working because the site has been down for quite a long time and other free software clients have dissapeared too. I compiled this source code but can't play anything

**Comment by SimonKagstrom** [comment]
Yes, I saw that as well. Probably spotify changed something - it worked when I last tried it a few weeks ago.
